src/plugins/appearence.js

```diff
@@ -1,5 +1,8 @@
 import prefix from '../prefix'
 
+/** Support old appearance props syntax.
```

Comment:
please use this style

/* Text...
 * 
 */

Unless you are going to generate jsdoc.